x-pack/test/apm_api_integration/tests/services/sorted_and_filtered_services.spec.ts

```diff
@@ -99,10 +99,10 @@ export default function ApiTest({ getService }: FtrProviderContext) {
 
       await synthtraceClient.index(eventsWithinTimerange.merge(eventsOutsideOfTimerange));
 
-      await Promise.all([
-        createAndRunApmMlJob({ environment: 'production', ml }),
-        createAndRunApmMlJob({ environment: 'development', ml }),
-      ]);
+      // Creating multiple ml jobs in parallel is causing this tests to be flaky
+      // https://github.com/elastic/elasticsearch/issues/36271
+      await createAndRunApmMlJob({ environment: 'production', ml });
+      await createAndRunApmMlJob({ environment: 'development', ml });
```

Comment:
Not sure, but I think there are other places that we create jobs in parallel as well. If so, can you update those too?

Reply:
You're right. I found `anomaly_charts` test as well. I'll update the PR.
Thanks!

Reply:
Good find @yngrdyn !!
Any idea why the [linked issue](https://github.com/elastic/elasticsearch/issues/36271) was closed yet, we are still running into this? Have we pinged ML about this?

Reply:
@sqren we discussed briefly in internal #machine-learning channel